pkg/kubeclient/client.go

```diff
@@ -0,0 +1,89 @@
+package kubeclient
+
+import (
+	"context"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+// Interface defines capability of the Kubernetes Client.
+type Interface interface {
```

Comment:
Don't we have an interface similar to that somewhere already? How is this interface better?

Reply:
Not that I know of.